### ISSUE TITLE
Fullscreen mouse position not accurate.

### ISSUE THREAD

**Issue by RensPitman** [RyanNielson/PixelCamera2D#3]
I'm using `Camera.main.ScreenToWorldPoint (Input.mousePosition)` to set a GameObject to the mouse position. In the editor it works but when playing the build on fullscreen, the GameObject doesn't follow the mouse correctly. 

What would be the correct way to fix this?

**Comment by RyanNielson** [comment]
Any chance you can upload a simple project that demonstrates that issue? It'd probably make it easier to diagnose.

**Comment by RensPitman** [comment]
https://github.com/Pearbook/NonAccurate/ Here you go. You only need to build it and run it on fullscreen and move your cursor toward the bottom-left corner of your screen (otherwise the gameobject will be outside of the screen)

**Comment by mwanninger** [comment]
@Pearbook I ran into the same issue. The issue appears to be something to do with how the world points from `ScreenToWorldPoint` are being returned in the Base Width x Base Height resolution of PixelCamera2D. For a workaround I did:

`float widthFactor = Screen.width / (float)(yourBaseWidth);`
`float heightFactor = Screen.height / (float)(yourBaseHeight);`
`float mouseAdjustment = widthFactor > heightFactor ? widthFactor : heightFactor;`

then when getting mouse position:
`var pos = Camera.main.ScreenToWorldPoint(Input.mousePosition / mouseAdjustment);`

Not the cleanest but seemed to work for now - have not yet tested on other resolutions to make sure it works; just tried on my 1920x1080 monitor running my game at 1080p.
